src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
+    * itemprops:
+        * value:`code` (syntax highlighting from Code node's language prop)
+* Collection
+    * `nav` `itemtype='schema:Collection'`
+    * itemprops:
+        * parts: `ol` of CreativeWorks/Articles (just title, author, and URIs/relative links?)
+* ContactPoint
+    * `address` `itemtype='schema:contactPoint'`
+    * itemprops:
+        * availableLanguages: `ul` of `data`? if not `p` (maybe unnecessary for styling)
+        * emails: `ol` of `a href`
+        * telephone: `a href`
+* CreativeWork:
+    * `div` `itemtype='schema:CreativeWork'`
+    * itemprops:
+        * authors: `ol` of `span` Person/Organization itemtypes.
+        * citations: `ol` of title/authors/`a href`
+        * content: `article`/`div` with `sections`
+        * dateCreated: `span time`
+        * dateModified: `span time`
+        * datePublished/date: `span time`
+        * editors: `ol` of `span` Person itemtypes
+        * funders: `ol` of `span` Person/Organization itemtypes
+        * isPartOf: `data`?
+        * licenses: `ol` of `a href`
+        * parts/hasPart: `ol` of figure/table/object/URI (maybe unnecessary for styling)
+        * publisher: `span` Person/Organization itemtype
+        * text: `data`? (maybe unnecessary for styling)
+        * title/headline: `h1`
+        * version: `span`
+* DataTable:
+    * `table` or `figure table` `itemtype='stencila:Datatable'`
+    * columns: probably want to convert these DatatableColumns into rows when encoding into HTML, so they're made up of `tr` and `td`/`th`. Maybe keep as `data`?
+        * values: `td`
+        * schema (items, uniqueItems): `data`? (maybe unnecessary for styling)
+        * name: `th`?
+* SoftwareSourceCode:
+    * codeRepository: `a href`
+    * codeSampleType: `data`?
+    * maintainers: `ol` of `span` Person/Organization itemtypes
+    * programmingLanguage: `data` or `span` (maybe unnecessary for styling)
+    * runtimePlatform: `ol` of `data` or `span`
+    * softwareRequirements: `ol` of `span` SoftwareSourceCode/SoftwareApplication
+    * targetProducts: `ol` of `span` SoftwareApplication
+
+### Schema nodes with direct mapping to semantic tags
+* Delete: `del`
+* Emphasis: `em`
+* Heading: 
+* ImageObject: `figure`
+    * thumbnail => `img`
+    * caption => `figcaption`
+* Link: `a`
+    * content
+    * target => href
+* List: `ol`, `ul` 
+  * ListItem => `li` (with `li` > `input type="checkbox"` option)
+* Quote
+* Strong => `strong`
+* Subscript => `sub`
+* Superscript => `sup`
+* Table => `table`. `table` > `tr`, or `table` > `thead`/`tfoot` > `tr`
+    * TableRow => `tr`.
+        * `kind`: header => `thead` > `tr`
+        * `kind`: footer => `tfoot` > `tr`
+    * TableCell => `th`, `td`
+        * `colspan`, `rowspan`
+        * `kind`: header => `th`
+        * `kind`: data => `td`
+* ThematicBreak: `hr`? (not sure if we should consider other options, such as splitting nodes by ThematicBreaks and grouping them into `section` tags instead)
```

Comment:
Yeah, kinda I like the idea of wrapping in `section` tags 👍
Though it does raise questions about decoding `section` tags and the extra logic we'd need to make sure we don't keep nesting `sections` when encoding multiple times as well as the different ways people can (ab)use them.

Reply:
Yeah, that might make things a bit more complicated. Maybe we'd want to include `itemtype='stencila:ThematicBreak'` either in the pre-break `<section>` tag, or in an `<hr>` in between `section` tags (and ignore `section` tags if they're previous/next siblings). Definitely seems simpler, but less semantic to stick with just `hr`.